ui/prometheus-plugin/src/utils/utils.ts

```diff
@@ -11,8 +11,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { replaceVariable } from '@perses-dev/plugin-system';
```

Comment:
This import was causing an issue with Jest:

```
FAIL src/utils/utils.test.ts
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'install')

      24 | import { getFormattedAxis } from '../utils';
      25 |
    > 26 | use([EChartsBarChart, GridComponent, DatasetComponent, TitleComponent, TooltipComponent, CanvasRenderer]);
         |    ^
      27 |
      28 | const BAR_WIN_WIDTH = 14;
      29 | const BAR_GAP = 6;

      at use (../node_modules/echarts/lib/extension.js:109:7)
      at ../node_modules/echarts/lib/extension.js:96:7
          at Array.forEach (<anonymous>)
      at each (../node_modules/echarts/node_modules/zrender/lib/core/util.js:205:13)
      at use (../node_modules/echarts/lib/extension.js:95:5)
      at Object.<anonymous> (../components/src/BarChart/BarChart.tsx:26:4)
      at Object.<anonymous> (../components/src/BarChart/index.ts:17:14)
      at Object.<anonymous> (../components/src/index.ts:18:14)
      at Object.<anonymous> (../plugin-system/src/components/CalculationSelector/CalculationSelector.tsx:24:21)
      at Object.<anonymous> (../plugin-system/src/components/CalculationSelector/index.ts:17:14)
      at Object.<anonymous> (../plugin-system/src/components/index.ts:17:14)
      at Object.<anonymous> (../plugin-system/src/index.ts:17:14)
      at Object.<anonymous> (src/utils/utils.ts:37:23)
      at Object.<anonymous> (src/utils/utils.test.ts:17:16)
```

This was due to the fact that Jest doesn't support tree shaking, about that see
- https://github.com/jestjs/jest/issues/11230
- https://dev.to/fogel/potential-issues-with-barrel-files-in-jest-1nkl

The location change here is just a way to work around the issue, it's actually surprising that we didnt face it earlier already..